app/uk/gov/hmrc/agentsubscriptionfrontend/config/frontendAppConfig.scala

```diff
@@ -39,42 +39,46 @@ trait AppConfig {
-}
+@Singleton
+class FrontendAppConfig @Inject() (val environment: Environment, val configuration: Configuration) extends AppConfig with ServicesConfig {
 
-object GGConfig extends StrictConfig {
-  private lazy val ggBaseUrl = loadConfig("authentication.government-gateway.sign-in.base-url")
-  lazy val ggSignInUrl: String = {
-    val ggSignInPath = loadConfig("authentication.government-gateway.sign-in.path")
-    s"$ggBaseUrl$ggSignInPath"
-  }
+  override val runModeConfiguration: Configuration = configuration
+  override protected def mode = environment.mode
 
-  lazy val checkAgencyStatusCallbackUrl: String = loadConfig("authentication.login-callback.url") +
-    routes.CheckAgencyController.showCheckAgencyStatus().url
-}
-
-@Singleton
-class FrontendAppConfig extends AppConfig with StrictConfig with ServicesConfig {
-  private lazy val contactHost = runModeConfiguration.getString(s"contact-frontend.host").getOrElse("")
-  private lazy val servicesAccountUrl = getConfString("agent-services-account-frontend.external-url", "")
-  private lazy val servicesAccountPath = getConfString("agent-services-account-frontend.start.path", "")
+  private val contactHost = getConfStringOrFail(s"contact-frontend.host")
+  private val servicesAccountUrl = getServicesConfStringOrFail("agent-services-account-frontend.external-url")
+  private val servicesAccountPath = getServicesConfStringOrFail("agent-services-account-frontend.start.path")
   private val contactFormServiceIdentifier = "AOSS"
 
-  override lazy val analyticsToken: String = loadConfig(s"google-analytics.token")
-  override lazy val analyticsHost: String = loadConfig(s"google-analytics.host")
-  override lazy val betaFeedbackUrl = s"$contactHost/contact/beta-feedback?service=$contactFormServiceIdentifier"
-  override lazy val betaFeedbackUnauthenticatedUrl = s"$contactHost/contact/beta-feedback-unauthenticated?service=$contactFormServiceIdentifier"
-  override lazy val reportAProblemPartialUrl = s"$contactHost/contact/problem_reports_ajax?service=$contactFormServiceIdentifier"
-  override lazy val reportAProblemNonJSUrl = s"$contactHost/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
-  override lazy val governmentGatewayUrl: String = loadConfig("government-gateway.url")
-  override lazy val blacklistedPostcodes: Set[String] =
+  override val analyticsToken: String = getConfStringOrFail(s"google-analytics.token")
+  override val analyticsHost: String = getConfStringOrFail(s"google-analytics.host")
+  override val betaFeedbackUrl = s"$contactHost/contact/beta-feedback?service=$contactFormServiceIdentifier"
+  override val betaFeedbackUnauthenticatedUrl = s"$contactHost/contact/beta-feedback-unauthenticated?service=$contactFormServiceIdentifier"
+  override val reportAProblemPartialUrl = s"$contactHost/contact/problem_reports_ajax?service=$contactFormServiceIdentifier"
```

Comment:
`reportAProblemPartialUrl` and `reportAProblemNonJSUrl` were added to the `application.conf`. 
Could use them here directly, or remove them from `application.conf`.